packages/app-spec-squad-chess/src/index.js

```diff
@@ -26,15 +27,14 @@ async function init() {
     return Object.assign(ps, p)
   })
   chess.registerPieces(pieces)
-  const startingPosition = JSON.parse(format.data).startingPosition
-  let turns = chess.generateTurns(startingPosition, 0)
 
+  const position = JSON.parse(format.data).startingPosition
+  let legalTurns = chess.generateTurns(position, 0)
```

Comment:
Let's also include this in the first rules.js function call -- maybe `createGame`.